packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -140,6 +140,7 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
                   target={this.state.refSelected}
                   setInitialFocus={true}
                   directionalHint={DirectionalHint.topRightEdge}
+                  id={'callout'}
```

Comment:
Don't hardcode an ID... what if you have 2 on the same page?

Just use `getId` in utilities to generate a unique ID and cache it in a member (this._calloutId) and use that.

Reply:
done